buildenv/docker/jdk14/s390x/ubuntu16/Dockerfile

```diff
@@ -87,7 +87,7 @@ RUN cd /root \
 
 # Download and install boot JDK from AdoptOpenJDK
 RUN cd /root \
-  && wget -O bootjdk12.tar.gz "https://api.adoptopenjdk.net/v2/binary/releases/openjdk12?openjdk_impl=openj9&os=linux&arch=s390x&release=latest&type=jdk" \
+  && wget -O bootjdk12.tar.gz https://api.adoptopenjdk.net/v3/binary/latest/12/ga/linux/s390x/jdk/openj9/normal/adoptopenjdk?project=jdk \
```

Comment:
This should get 13, not 12.

Reply:
Fixed in [force-push](https://github.com/eclipse/openj9/compare/eb9921d756fb42fa732cc9ccf4402009a13f5d52..2b597d6ed80b336e8cab7295a59f670602b5e097).